doc/nrf/ug_matter_gs_kconfig.rst

```diff
@@ -0,0 +1,24 @@
+.. _ug_matter_gs_kconfig:
+.. _ug_matter_configuring_protocol:
+
+Enabling Matter in Kconfig
+##########################
+
+To use the Matter protocol, set the following Kconfig options:
+
+* :kconfig:option:`CONFIG_CHIP` - This option enables the Matter protocol stack and other associated Kconfig options, including :kconfig:option:`CONFIG_CHIP_ENABLE_DNSSD_SRP` that is required for the discovery of the Matter device using DNS-SD.
+* :kconfig:option:`CONFIG_CHIP_PROJECT_CONFIG` - This option defines the path to the configuration file that specifies Vendor ID, Product ID, and other project-specific Matter settings.
```

Comment:
Currently, most settings that used to be configured with this file can be configured via Kconfig, so there's no need to mention it here. I would move it to the advanced configuration part with description: "This option defines the path to the configuration file that contains project-specific Matter settings in the form of C preprocessor macros, that cannot be altered using dedicated Kconfig options"